macosx/ReadMe.md

```diff
@@ -0,0 +1,656 @@
+# title: GRASS GIS Mac OS X Build
```

Comment:
| :exclamation:  Instructions are at present not up-to-date. Update in progress. |
|--------------------------------------------------------------------------------|